pkg/api/types.go

```diff
@@ -926,6 +914,12 @@ const (
 	// Status code 422
 	StatusReasonInvalid StatusReason = "Invalid"
 
+	// StatusReasonTimeout means that the request could not be completed within the given time.
```

Comment:
IIUC, the lists need to match in the versioned types.go files. I don't understand why this list contains so many constants that the others do not, and why the others contain at least one that this does not.

@smarterclayton Is this just a bug?